packages/protocol/contracts/libs/LibTrieProof.sol

```diff
@@ -18,22 +18,21 @@ library LibTrieProof {
     uint256 private constant ACCOUNT_FIELD_INDEX_STORAGE_HASH = 2;
 
     error LTP_INVALID_ACCOUNT_PROOF();
+    error LTP_INVALID_INCLUSION_PROOF();
 
     /**
      * Verifies that the value of a slot in the storage of an account is value.
      *
      * @param stateRoot The merkle root of state tree.
      * @param addr The address of contract.
      * @param slot The slot in the contract.
-     * @param value The value to be verified.
      * @param mkproof The proof obtained by encoding storage proof.
      * @return verified The verification result.
      */
     function verifyFullMerkleProof(
         bytes32 stateRoot,
         address addr,
         bytes32 slot,
-        bytes32 value,
```

Comment:
I guess could still take `bytes memory value` without extra complications to allow the lib to be used to check other values.

Reply:
Sure thing i thought the same too, tho i tried to play around with correct conversions.

So in SignalService we do: hex"01", it translates to bytes32 and by default already it is wrong (padding and trailing zeros)
0x0100000000000000000000000000000000000000000000000000000000000000

How can we overcome this easiest ?

Reply:
But if you change `bytes32 value` to `bytes memory value` in `verifyFullMerkleProof` there won't be a conversion to bytes32 first right?

Reply:
should be, yes, totally missed that. 🤣  will do real quick.